### DISCUSSION TITLE
How can I use "ZipDirectory" to zip output files into publish directory instead of "copy files" at the end of Publish?

### DISCUSSION THREAD

**Discussion by Flithor** [dotnet/msbuild#8584] [Q&A]
title
Ask a `.csproj` solution.

**Comment by JanKrivanek** [comment]
Publish is a https://github.com/dotnet/sdk feature, so you might get more tailored help there. But did you try [Zip Deploy](https://github.com/dotnet/sdk/blob/48b70ad9fb385b6cc08e344e16e9790544e9e310/src/WebSdk/README.md?plain=1#L118)?
